sysconfigs/tsa/packages.yaml

```diff
@@ -42,6 +50,10 @@ packages:
   perl:
     paths: 
       perl@5.16.3: /usr
+  pgi:
+    buildable: false
+    modules:
+      pgi@19.9: PrgEnv-pgi/19.9
```

Comment:
this has not been working so far with cosmo, because during the build we need to change environment (gcc for dycore) and fortran for PGI, this did not work well with PrgEnv

Reply:
Thought modules were the preferred way. I'll change it back then. Note however, that PrgEnv seems to be the only way to use PGI on daint. Install does not work without loaded module.